tests/indent/python_spec.lua

```diff
@@ -26,8 +26,8 @@ describe("indent Python:", function()
     run:new_line("basic_blocks.py", { on_line = 1, text = "wait,", indent = 4 })
     run:new_line("basic_blocks.py", { on_line = 6, text = "x += 1", indent = 4 })
     run:new_line("basic_blocks.py", { on_line = 10, text = "x += 1", indent = 8 })
-    run:new_line("basic_blocks.py", { on_line = 7, text = "x += 1", indent = 4 }, "7, after last line of a block")
-    run:new_line("basic_blocks.py", { on_line = 11, text = "x += 1", indent = 8 }, "11, after last line of a block")
+    run:new_line("basic_blocks.py", { on_line = 7, text = "x += 1", indent = 0 })
+    run:new_line("basic_blocks.py", { on_line = 11, text = "x += 1", indent = 4 })
```

Comment:
Reason: `pass` is marked as `@return` node. So if the cursor is on a line with `pass`, entering newline should not keep indentation, it should dedent instead.